client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -386,6 +395,30 @@ const CancelJetpackForm: React.FC< Props > = ( {
 	const renderCurrentStep = () => {
 		const productName = getName( purchase );
 
+		if ( steps.CANCEL_CONFIRM_STEP === cancellationStep ) {
+			return (
+				<>
+					<FormattedHeader
+						headerText={ translate( 'Confirm removal' ) }
+						subHeaderText={
+							/* Translators: %(percentDiscount)d%% should be a percentage like 15% or 20% */
+							translate(
+								'We’re sorry to see you go. Click Remove subscription to confirm and remove %(productName)s from your account.',
+								{
+									args: {
+										productName,
+									},
+								}
+							)
+						}
+						align="center"
+						isSecondary={ true }
+					/>
+					<p></p>
```

Comment:
What is this <p> tag for exactly?

Reply:
It was a leftover from when I was originally composing the message - thanks for catching!